resources/js/pages/GroupPage.vue

```diff
@@ -6,90 +6,51 @@
       </div>
       <ViewGroup
         v-if="!editing && group"
-        v-model:editing="editing"
+        v-model:is-editing="editing"
```

Comment:
Sorry, I missed this comment before merging.

the hyphen thing happens sometimes with vs code autocomplete, which I typically ignore since it's supposed to fixed on save. I'm guessing the `:` prevented to fix on save to work properly.  I'll make the change.